client/src/bundles/bundle.scss

```diff
@@ -8,6 +8,7 @@
 @import '../styles/mfa';
 
 // Components
+@import '../components/Register/Introduction';
```

Comment:
My eye twitched slightly - can you move this down with the other Register components?